src/renderer/components/dex/OrderHistory.vue

```diff
@@ -107,7 +108,7 @@ export default {
     this.loadOrders();
 
     loadOrdersIntervalId = setInterval(() => {
-      this.loadOrders();
+      this.loadOrders(true);
```

Comment:
Another great place to abstract to make it clear what `true` means. 

`loadOrdersSilently`